app/src/main/java/org/sil/storyproducer/model/Workspace.kt

```diff
@@ -54,6 +59,9 @@ object Workspace {
     const val URL_FOR_TEMPLATES_PLACE_HOLDER = "URL_FOR_TEMPLATES_PLACE_HOLDER"
     // End Issue #571
 
+    val NUM_BYTES_NEEDED_FOR_DEMO_STORY = 1024 * 1024 * 10L;
```

Comment:
Need at least 100 MB for demo project - should we check for 500 MB to prevent user running out of storage because of SP?

Reply:
Done but if 1GB is not free internally then 200MB is used as the minimum free space required.  This is checked initially in internal backed memory then in external (sd card) memory.